src/js/grapl-cdk/swarm/swarm_setup.py

```diff
@@ -80,6 +81,21 @@ def _get_command_result(ssm: Any, command_id: str, instance_id: str) -> str:
 CW_DISK_USAGE_METRIC_NAME = "disk_used_percent"
 
 
+def _find_operational_alarms_arn(
+    prefix: str,
+    sns: Optional[SNSClient] = None,
+) -> str:
```

Comment:
this is all done because, well, constructing my own ARN is a pain in the ass - how the hell do i find account id and region with boto3, etc

Reply:
I think hitting the instance metadata is the way to go. This is how we do it when we initialize the docker swarm manager: https://github.com/grapl-security/grapl/blob/staging/src/js/grapl-cdk/swarm/swarm_init.py#L7-L12